common/core/desktop/src/kmx/kmx_processor.cpp

```diff
@@ -75,43 +103,99 @@ kmx_processor::update_option(
   return option(scope, key, value);
 }
 
+bool kmx_processor::queue_action(
+  km_kbp_action_item const * action_item
+) {
+  DebugLog("Action type is [%d].\n", action_item->type);
+  switch (action_item->type) {
+  case KM_KBP_IT_END:
+    // error should not queue empty item
```

Comment:
Let's throw a debug log statement here given it's not a thing that should be done and probably indicates a logic bug somewhere...